tests/end-to-end/simple/app.go

```diff
@@ -152,10 +152,10 @@ func NewNoctStack(scope constructs.Construct, id string, props NoctStackProps) *
 	}
 }
 
-/// ifCondition is a helper function that replicates the ternary
-/// operator that can be found in other languages. It is conceptually
-/// equivalent to writing `cond ? whenTrue : whenFalse`, meaning it
-/// returns `whenTrue` if `cond` is `true`, and `whenFalse` otherwise.
+// / ifCondition is a helper function that replicates the ternary
+// / operator that can be found in other languages. It is conceptually
+// / equivalent to writing `cond ? whenTrue : whenFalse`, meaning it
+// / returns `whenTrue` if `cond` is `true`, and `whenFalse` otherwise.
```

Comment:
I think this means it's no longer a specific type of doc comment, so we should undo this one?